### DISCUSSION TITLE
Community Surveys

### DISCUSSION THREAD

**Discussion by michallawson** [HCL-TECH-SOFTWARE/connections-automation#66] [Ideas]
Would it be possible for you to create ansible scripts to ease installation of Community Surveys?

Regards

Michael

**Comment by dejo1307** [comment]
Hi Michael,

Let me take a look how much effort would be needed. I don't see why not, my only concern would be (as I never took a look into CS myself) if this is something that was never 'intended' to be automated (like is it possible to do it in some silent mode). 

Best regards,
Dejan

**Comment by michallawson** [comment]
Dejan,

Thank you.

I started to look at installing it manually, but it needs changes to the DB and as I am not sure what all of the DB2 users do or what their roles are I thought it would be safer if there was an Ansible script, so it is consistent with the Connections and Docs installation.

It is not critical.

Regards
Michael
